src/test_driver/esp32/sdkconfig.defaults

```diff
@@ -31,6 +31,7 @@ CONFIG_ESP_MAIN_TASK_STACK_SIZE=32768
 
 #enable BT
 CONFIG_BT_ENABLED=y
+CONFIG_BT_NIMBLE_ENABLED=y
```

Comment:
Why do you need to explicitly add this? Is NimBLE not enabled by default, when BT is enabled?

Reply:
Default is bluedroid, needs this to fix the CI.

Reply:
https://github.com/espressif/esp-idf/blob/master/components/bt/Kconfig#L12